src/main/kotlin/io/billie/organisations/orders/domain/OrderAmount.kt

```diff
@@ -0,0 +1,12 @@
+package io.billie.organisations.orders.domain
+
+import java.math.BigDecimal
+
+/**
+ * Domain object represents order amount, should always be positive
+ */
+data class OrderAmount(val amount: BigDecimal) {
+    init {
+        require(amount.compareTo(BigDecimal.ZERO) > 0) { "Order amount must be positive" }
```

Comment:
Order amount and also shipment amount domain objects ensure that it always receives positive value